03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,15 @@
+    1. Quickly explain how middleware is used in an Express application.
+    -middleware takes the req and res, performs a function and then either ends the cycle or passed on to the next middleware function (often called next()).
+   
+    2. What does CRUD stand for?
+    -create
+    -read
+    -update
+    -delete
+    
+    3.You are careful to store your sensitive information in a .env file.
+        a.How do you get your app to access the information in the .env file?
+        -Require the env in another file and then use the variable named in the env file to process the variable. 
```

Comment:
This is pretty close - there is a subtle difference here, in that you are requiring a library (`dotenv`) that actually reads the `.env` file and loads it into the `process.env` environment variable. `process` is a special global in node that contains information about the process the application is running in, one of which is the environment (`env`) that includes the variables that were present when the app was started, and also any that are added to it like `dontenv` does.